src/compile/model.ts

```diff
@@ -553,10 +553,9 @@ export abstract class ModelWithField extends Model {
   public field(channel: SingleDefChannel, opt: FieldRefOption = {}) {
     const fieldDef = this.fieldDef(channel);
 
-
-    if (fieldDef.bin) { // bin has default suffix that depends on scaleType
+    if (fieldDef.bin) {
       opt = extend({
-        binSuffix: this.hasDiscreteDomain(channel) ? 'range' : 'start'
+        binSuffix: 'start'
```

Comment:
We can explicitly call it but I don't see why you would ever want range unless it's used for labeling things.